README.md

```diff
@@ -1,2 +1,42 @@
-# morse_code_with_ruby
-The project is about creating a project that will be able to decode Morse code.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Morse code decoder
+
+> This project is to create a program that will automate the decoding of morse code.
+
+
+## Built With
+
+- Ruby

```

Comment:
- [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.